yarn.lock

```diff
@@ -698,6 +698,16 @@ ajv@^5.1.0:
     json-schema-traverse "^0.3.0"
     json-stable-stringify "^1.0.1"
 
+ajv@^6.1.0:
```

Comment:
`ajv` and `loader-utils` can be resolved to upper versions. `yarn-deduplicate` can do that.